.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+<!--- Add a pull request title above in this format -->
+<!--- real example: 'RESTWS-738 Remove concept property setters from ObsResource classes' -->
+<!--- 'RESTWS-JiraIssueNumber JiraIssueTitle' -->
```

Comment:
Was this also part of the changes need to make it compile on Java 17?

Reply:
I tried to kill two birds with one stone :)

Reply:
Did you get a chance to look at this? https://opensource.com/article/18/6/anatomy-perfect-pull-request